open-sphere-base/core/src/main/java/io/opensphere/core/util/fx/FXUtilities.java

```diff
@@ -453,8 +454,15 @@ public static Color fromAwtColor(java.awt.Color awtColor)
      */
     public static java.awt.Color toAwtColor(Color color)
     {
-        // TODO add opacity
-        return new java.awt.Color(color.hashCode() >> 8);
+        java.awt.Color awtColor = null;
+
+        if (color != null)
```

Comment:
what do we do if the supplied color is null? Default to white? Can that be explicit?